From 0eccee18822e60393c8a4a9b99a3c80d2b1275d9 Mon Sep 17 00:00:00 2001
From: Alexey Tikhonov <atikhono@redhat.com>
Date: Mon, 14 Jun 2021 21:47:52 +0200
Subject: [PATCH 03/16] secrets: reduce log severity in local_db_create() in
 case entry already exists since this is expected during normal oprations.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
Reviewed-by: Sumit Bose <sbose@redhat.com>
---
 src/util/secrets/secrets.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/util/secrets/secrets.c b/src/util/secrets/secrets.c
index 6e99e291dd355cc69b0d872f53624ca3446e18ad..f12b615f8ac8f929969d328db41876070ccc75c5 100644
--- a/src/util/secrets/secrets.c
+++ b/src/util/secrets/secrets.c
@@ -476,7 +476,7 @@ static int local_db_create(struct sss_sec_req *req)
     ret = ldb_add(req->sctx->ldb, msg);
     if (ret != LDB_SUCCESS) {
         if (ret == LDB_ERR_ENTRY_ALREADY_EXISTS) {
-            DEBUG(SSSDBG_OP_FAILURE,
+            DEBUG(SSSDBG_FUNC_DATA,
                   "Secret %s already exists\n", ldb_dn_get_linearized(msg->dn));
         } else {
             DEBUG(SSSDBG_CRIT_FAILURE,
-- 
2.20.1

